vcr-gha.yml

```diff
@@ -4,6 +4,7 @@ instance:
   name: dev
   runtime: nodejs22
   region: aws.euw1
+  build-script: './vcrBuild.sh'
```

Comment:
This is the addition that we need to have the app build on VCR, same as we do for local deployment.
This is a deviation from where we were building on GHA runner previously. 
We can also do that as one option but @dwivedisachin ok'd this approach.
The other approach is basically `rm` the `.vcrignore` file in the gha deploy script